hw1_iphone/styles/styles .css

```diff
@@ -0,0 +1,335 @@
+
+  --text-color-default: #c8c7cc;
+  --bg-color-default: #f9f9f9;
+  --screen-color-default: #0c0d0b;
+
+  --transition-duration: 0.5s;
+  --opacity-default: 0.8;
+}
+
+.bottom-bar_btn,
+.camera-btn_img,
+.slider-text,
+.slider,
+.time,
+.battery,
+.date,
+.top-btn,
+.signal-icons {
+  visibility: hidden;
+}
+
+.container {
+  background: url("/hw1_iphone/images/background.png") no-repeat;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.iphone {
+  border: 3px solid transparent;
+  border-radius: 36px;
+  width: 290px;
+  height: 596px;
+  position: relative;
+  box-shadow: rgb(0 0 0 / 97%) -8px 2px 34px, rgb(0 0 0 / 47%) -22px 6px 18px;
+  background: linear-gradient(white, white) padding-box,
+    linear-gradient(to top left, var(--beige-default), #a98d76) border-box;
+  display: block;
+  margin: auto;
+  color: var(--white);
+}
+
+.screen {
+  position: absolute;
+  background-size: cover;
+  width: 260px;
+  height: 453px;
+  background-color: var(--screen-color-default);
+  margin: 60px 13px;
+  border: 2px solid var(--screen-color-default);
+  box-shadow: rgb(0 0 0 / 15%) 6px 17px 31px;
+  border-radius: 3px;
+}
+
+.screen:hover {
+  position: absolute;
+  background-size: cover;
+  width: 260px;
+  height: 453px;
+  background-image: url("/hw1_iphone/images/content/background-screen.png");
+  margin: 60px 13px;
+  border: 2px solid var(--screen-color-default);
+  box-shadow: rgb(0 0 0 / 15%) 6px 17px 31px;
+  border-radius: 3px;
+}
+
+.screen:hover .top-btn {
+  visibility: visible;
+}
+
+.tools {
+  z-index: 15;
+  position: absolute;
+  width: 223px;
+  height: 50px;
+  margin-left: 30px;
+  margin-top: 7px;
+}
+
+.front-camera_left {
+  position: absolute;
+  border-radius: 50%;
+  border: 4px solid rgb(28, 30, 20);
+  background: rgb(15, 15, 14);
+  top: 18px;
+  left: 68px;
+}
+
+.front-camera_top {
+  position: absolute;
+  border-radius: 50%;
+  border: 0.2rem solid #1c1e14;
+  background: #0f0f0e;
+  top: 3px;
+  left: 110px;
+}
+
+.microphone {
+  position: absolute;
+  top: 20px;
+  right: 87px;
+  width: 42px;
+  height: 3px;
+  border-radius: 0.2rem;
+  background: #1d1312;
+  border: 1px solid #9f9796;
+}
+
+.bottom-btn {
+  position: absolute;
+  border-radius: 50%;
+  border: 3px solid transparent;
+  background: linear-gradient(white, white) padding-box,
+    linear-gradient(to top left, #3c3628, var(--white)) border-box;
+  top: 533px;
+  left: 124px;
+  width: 45px;
+  height: 43px;
+  cursor: pointer;
+  opacity: var(--opacity-default: );
+}
+
+.top-bar {
+  position: absolute;
+  width: 251px;
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+  align-items: center;
+  margin: 2px 5px;
+  opacity: var(--opacity-default: );
+}
+
+.signal-icons {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-flow: row wrap;
+}
+
+.screen:hover .signal-icons {
```

Comment:
Happy to you I have hovered the screen. Otherwise I could decide that it is black